code/ui/tttpanel/TTTPanel.cs

```diff
@@ -0,0 +1,24 @@
+using Sandbox.UI;
+
+namespace TTTReborn.UI
+{
+    public class TTTPanel : Panel
+    {
+        public bool IsShowing
+        {
+            get => _isShowing;
+            set
+            {
+                _isShowing = value;
+
+                SetClass("hide", !_isShowing);
+            }
+        }
+        private bool _isShowing = false;
+
+        public TTTPanel() : base()
+        {
+            IsShowing = false;
+        }
+    }
+}
```

Comment:
Nice :D, do you want to add an issue whenever this is merged to add this to all panels?

Reply:
You mean part of a refactoring like the still pending https://github.com/TTTReborn/tttreborn/pull/139 :P

Reply:
https://github.com/TTTReborn/tttreborn/issues/168